hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenPauseSpecs.java

```diff
@@ -273,7 +275,7 @@ private HapiSpec pausedNonFungibleUniqueCannotBeUsed() {
                         tokenCreate("failedTokenCreate")
                                 .treasury(TOKEN_TREASURY)
                                 .withCustom(fixedHtsFee(1, uniqueToken, firstUser))
-                                .hasKnownStatus(INVALID_TOKEN_ID_IN_CUSTOM_FEES),
+                                .hasKnownStatus(CUSTOM_FEE_DENOMINATION_MUST_BE_FUNGIBLE_COMMON),
```

Comment:
This asserion was updated because the status `CUSTOM_FEE_DENOMINATION_MUST_BE_FUNGIBLE_COMMON` is more descriptive and it was added to the code some time after the test was originally written.